components/StandardApp.jsx

```diff
@@ -140,6 +141,12 @@ class AppInitComponent extends React.Component {
                 }
             });
         });
+        const task = ConfigUtils.getConfigProp("startupTask")
+        console.log(task);
```

Comment:
You'll want to drop this line

Reply:
well ok :)  => Done 

sorry for this left over.